src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurCashAdvancePdpFeedFileServiceImpl.java

```diff
@@ -172,6 +185,68 @@ private PdpFeedTrailerEntry buildPdpFeedTrailerEntry(int totalDetailRecordCount,
         return trailer;
     }
 
+    private void updateReportDataPdpFeedFileCountAndAmountTotals(ConcurRequestExtractBatchReportData reportData, ConcurRequestExtractRequestDetailFileLine detailFileLine) {
+        int count = reportData.getCashAdvancesProcessedInPdp().getRecordCount();
```

Comment:
Just offering a thought, how would it be if the functions like updateReportDataPdpFeedFileCountAndAmountTotals were reportData object?